### PR TITLE
feat(Discover): add feature flag for sberbank tv url

### PR DESCRIPTION
**Pull request by TyaglovOV**

### CODE REVIEW

File: docs/feature-flags.schema.json [hunk 1]
```diff
@@ -7,6 +7,12 @@
       "default": true,
       "description": "if true = discover tab is enable for users"
     },
+    "discover.tvUrl": {
+      "type": "string",
+      "title": "Url of discover tab iframe content",
+      "default": "https://sberbanktv.ru/",
```
Comment (nkt) [inline docs/feature-flags.schema.json]: Remove default value

File: docs/feature-flags.schema.json [hunk 2]
```diff
@@ -7,6 +7,12 @@
...
+      "description": "If url is presented, will render sberbankTV page with videos in discovery tab"
```
Comment (nkt) [inline docs/feature-flags.schema.json]: Update description after remove default value

File: docs/feature-flags.schema.json (hunk 1 above)
Comment (TyaglovOV) [reply docs/feature-flags.schema.json]: yep

File: docs/feature-flags.schema.json (hunk 2 above)
Comment (TyaglovOV) [reply docs/feature-flags.schema.json]: yep

File: docs/feature-flags.schema.json [hunk 3]
```diff
@@ -7,6 +7,11 @@
       "default": true,
       "description": "if true = discover tab is enable for users"
     },
+    "discover.tvUrl": {
```
Comment (olegshilov) [inline docs/feature-flags.schema.json]:
```suggestion
    "discover.iframeURL": {
```

**Pull request closed without merging**